R/covr.R

```diff
@@ -45,8 +45,10 @@ reset_traces <- function() {
 }
 
 save_trace <- function(directory) {
-  tmp_file <- temp_file("covr_trace_", tmpdir = directory)
-  saveRDS(.counters, file = tmp_file)
+  counters_tmp_file <- temp_file("covr_trace_", tmpdir = directory)
```

Comment:
rename this is to just tmp_file so the diff is a bit smaller

Reply:
done